modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -46,6 +46,13 @@ spec:
 When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0-9_]+$` in the `resourceName`.
 
 <4> The node selector specifies the nodes to configure. Only SR-IOV network devices on the selected nodes are configured. The SR-IOV Container Network Interface (CNI) plugin and device plugin are deployed on selected nodes only.
++
+[IMPORTANT]
+====
+If a node is in an unhealthy machine config pool, the process of applying node network configuration policies to all targeted nodes in the cluster will pause until the machine config pool returns to a healthy state. This interruption occurs because the SR-IOV Network Operator checks that a node's MCP is not in a `Degraded` or `Updating` state before applying a policy and the SR-IOV Network Operator applies policies to nodes in sequence.
```

Comment:
```suggestion
If a node is in an unhealthy machine config pool, the process of applying node network configuration policies to all targeted nodes in the cluster pauses until the machine config pool returns to a healthy state. This interruption occurs because the SR-IOV Network Operator checks that a node's MCP is not in a `Degraded` or `Updating` state before applying a policy and the SR-IOV Network Operator applies policies to nodes in sequence.
```
https://redhat-documentation.github.io/supplementary-style-guide/#will ==> 
`Do not use future tense unless it is absolutely necessary.`